lbz/resource.py

```diff
@@ -101,24 +85,54 @@ def _get_user(self, headers: CIMultiDict) -> Union[None, User]:
         return None
 
     def _post_request_hook(self) -> None:
-        """
+        """*/
         Makes the post_request_hook run-time friendly.
         """
         try:
             self.post_request_hook()
         except Exception as err:  # pylint: disable=broad-except
             logger.exception(err)
 
+    @deprecated
     def pre_request_hook(self) -> None:
         """
         Place to configure pre request hooks.
+
+        deprecated - please use pre_handle
         """
 
+    @deprecated
     def post_request_hook(self) -> None:
         """
         Place to configure post request hooks.
+
+        deprecated - please use post_handle
         """
 
+    def handle(self) -> Response:
```

Comment:
I liked the previous order of methods where `handle()` is treated as the most important and is declared above the helpers 😉